lib/k8y/kubeconfig/cluster.rb

```diff
@@ -23,7 +23,7 @@ def initialize(name:, insecure_skip_tls_verify:, certificate_authority:, certifi
         @insecure_skip_tls_verify = insecure_skip_tls_verify
         @certificate_authority = certificate_authority
         @certificate_authority_data = certificate_authority_data
-        @server = URI.parse(server)
+        @server = server
```

Comment:
URI parsing too early just leads to too much heartache when it comes time to perform joins on prefixes and paths. Deferring URI parsing seems the wisest move for now